cloudigrade/internal/prometheus.py

```diff
@@ -83,6 +88,47 @@ def _initialize_celery_queue_length_metrics(self):
                     partial(redis.execute_command, "llen", [queue_name])
                 )
 
+    def _initialize_db_table_size_metrics(self):
+        """Initialize Database table size metrics."""
+        gauge = Gauge(
+            DB_TABLE_SIZE_METRIC_NAME,
+            "The size of the tables in bytes.",
+            labelnames=["table_name"],
+        )
+        self._gauge_metrics[DB_TABLE_SIZE_METRIC_NAME] = gauge
+        curs = connection.cursor()
+        curs.execute(
+            "SELECT tablename AS table_name,"
+            " pg_total_relation_size(quote_ident(tablename)) AS table_size"
+            " FROM (SELECT * FROM pg_catalog.pg_tables"
+            " WHERE schemaname = 'public') t"
+            " ORDER by table_name"
+        )
```

Comment:
the method here and for the number of rows in the lower method are PG specific, thus failure in pytest, where we're running with sqllite.  We will need to differentiate backend db type, if PG do the logic here, otherwise, go with more generic db/agnostic queries.

Specific error here:

```
  File "/__w/cloudigrade/cloudigrade/.tox/py39/lib/python3.9/site-packages/django/db/backends/sqlite3/base.py", line 421, in execute
    return Database.Cursor.execute(self, query)
django.db.utils.OperationalError: no such table: pg_catalog.pg_tables
```